bugbot/rules/webcompat_sightline.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatSightline(BzCleaner):
+    normal_changes_max = 1000
+    whiteboard_entry = "[webcompat:sightline]"
```

Comment:
I wouldn't do that; it's only really used inside the class so this approach makes it clearer in my mind. But ofc we can change it if it will make it closer to project norms.